### PR TITLE
skr

### PR DESCRIPTION
**Pull request by Haocen2004**
command
tab
and
more

### CODE REVIEW

**Review by misterplus — CHANGES_REQUESTED** [review]
.gitignore credits changelog
这三个不需要

File: src/main/java/plus/misterplus/ivrench/InvertedEnchantments.java [hunk 1]
```diff
@@ -14,6 +18,11 @@
     public static final String VERSION = "1.0.3";
 
     private static Logger logger = LogManager.getLogger(MOD_NAME);
+    public final static CreativeTab TAB = new CreativeTab();
```
Comment (misterplus) [inline src/main/java/plus/misterplus/ivrench/InvertedEnchantments.java]: change to
`public final static CreativeTab MOD_TAB = new CreativeTab();`

File: src/main/resources/assets/ivrench/lang/en_us.lang [hunk 2]
```diff
@@ -27,4 +27,6 @@ enchantment.aqua=Aqua
 enchantment.finity=Finity
 enchantment.lureless=Lureless
 enchantment.unmending=Unmending
-enchantment.vanishing=Curse of Instant Vanishing
\ No newline at end of file
+enchantment.vanishing=Curse of Instant Vanishing
+
+itemGroup.ivrench=Ivrench Book
```
Comment (misterplus) [inline src/main/resources/assets/ivrench/lang/en_us.lang]: Change to "Inverted Enchantment"

**Review by misterplus — CHANGES_REQUESTED** [review]

File: src/main/java/plus/misterplus/ivrench/InvertedEnchantments.java (hunk 1 above)
Comment (Haocen2004) [reply src/main/java/plus/misterplus/ivrench/InvertedEnchantments.java]: got it

**Comment by Haocen2004** [comment]
ok

**Pull request merged**